### PR TITLE
Use 'et format' in CI. Check formatting of all files in CI

### PR DESCRIPTION
**Pull request by zanderso**
This PR changes the format check on CI to use the command added in https://github.com/flutter/engine/pull/50747.

Additionally, while making this change, I noticed that the CI check was not checking the formatting of all files, and that as a result, files were present in the repo with incorrect formatting. I have fixed the formatting and fixed the check to always check all files.

### CODE REVIEW

**Review by johnmccutchan — APPROVED** [review]
RSLGTM

File: impeller/renderer/backend/gles/proc_table_gles.h [hunk 1]
```diff
@@ -37,12 +37,10 @@ struct AutoErrorCheck {
...
       if (GLErrorIsFatal(error)) {
         FML_LOG(FATAL) << "Fatal GL Error " << GLErrorToString(error) << "("
-                       << error << ")"
-                       << " encountered on call to " << name;
+                       << error << ")" << " encountered on call to " << name;
       } else {
         FML_LOG(ERROR) << "GL Error " << GLErrorToString(error) << "(" << error
-                       << ")"
-                       << " encountered on call to " << name;
+                       << ")" << " encountered on call to " << name;
```
Comment (loic-sharma) [inline impeller/renderer/backend/gles/proc_table_gles.h:43]: Super nit, feel free to skip:

```suggestion
                       << ") encountered on call to " << name;
```

File: lib/ui/window/viewport_metrics.cc [hunk 2]
```diff
@@ -95,15 +95,14 @@ bool operator==(const ViewportMetrics& a, const ViewportMetrics& b) {
...
-     << a.physical_padding_left << "L] "
+  os << "DPR: " << a.device_pixel_ratio << " " << "Size: [" << a.physical_width
+     << "W " << a.physical_height << "H] " << "Padding: ["
+     << a.physical_padding_top << "T " << a.physical_padding_right << "R "
+     << a.physical_padding_bottom << "B " << a.physical_padding_left << "L] "
      << "Insets: [" << a.physical_view_inset_top << "T "
      << a.physical_view_inset_right << "R " << a.physical_view_inset_bottom
-     << "B " << a.physical_view_inset_left << "L] "
-     << "Gesture Insets: [" << a.physical_system_gesture_inset_top << "T "
+     << "B " << a.physical_view_inset_left << "L] " << "Gesture Insets: ["
```
Comment (loic-sharma) [inline lib/ui/window/viewport_metrics.cc:104]:
```suggestion
     << "B " << a.physical_view_inset_left << "L] Gesture Insets: ["
```

File: shell/platform/darwin/ios/framework/Source/FlutterDartVMServicePublisher.mm [hunk 3]
```diff
@@ -129,8 +129,7 @@ - (void)publishServiceProtocolPort:(NSURL*)url {
                    << "as hot reload and DevTools. To make your Flutter app or module "
                    << "attachable and debuggable, add a '" << registrationType << "' value "
                    << "to the 'NSBonjourServices' key in your Info.plist for the Debug/"
-                   << "Profile configurations. "
-                   << "For more information, see "
+                   << "Profile configurations. " << "For more information, see "
```
Comment (loic-sharma) [inline shell/platform/darwin/ios/framework/Source/FlutterDartVMServicePublisher.mm:132]:
```suggestion
                   << "Profile configurations. For more information, see "
```

File: shell/platform/fuchsia/flutter/focus_delegate_unittests.cc [hunk 4]
```diff
@@ -143,8 +143,7 @@ TEST_F(FocusDelegateTest, RequestFocusTest) {
 
   // Create the platform message request.
   std::ostringstream message;
-  message << "{"
-          << "    \"method\":\"View.focus.request\","
+  message << "{" << "    \"method\":\"View.focus.request\","
```
Comment (loic-sharma) [inline shell/platform/fuchsia/flutter/focus_delegate_unittests.cc:146]:
```suggestion
  message << "{    \"method\":\"View.focus.request\","
```

File: shell/platform/fuchsia/flutter/platform_view.cc [hunk 5]
```diff
@@ -381,15 +381,12 @@ void PlatformView::OnChildViewStatus(
   FML_DCHECK(child_view_info_.count(content_id) == 1);
 
   std::ostringstream out;
-  out << "{"
-      << "\"method\":\"View.viewStateChanged\","
-      << "\"args\":{"
+  out << "{" << "\"method\":\"View.viewStateChanged\"," << "\"args\":{"
```
Comment (loic-sharma) [inline shell/platform/fuchsia/flutter/platform_view.cc:384]: Same comment here for JSON string readability

File: shell/platform/fuchsia/flutter/pointer_injector_delegate_unittest.cc [hunk 6]
```diff
@@ -108,11 +108,9 @@ class PlatformMessageBuilder {
 
   rapidjson::Value Build() {
     std::ostringstream message;
-    message << "{"
-            << "    \"method\":\""
+    message << "{" << "    \"method\":\""
```
Comment (loic-sharma) [inline shell/platform/fuchsia/flutter/pointer_injector_delegate_unittest.cc:111]: Same comment for JSON string readability in this file

File: shell/platform/fuchsia/flutter/tests/platform_view_unittest.cc [hunk 7]
```diff
@@ -803,14 +803,10 @@ TEST_F(PlatformViewTests, CreateViewTest) {
...
   std::ostringstream create_view_message;
-  create_view_message << "{"
-                      << "  \"method\":\"View.create\","
-                      << "  \"args\":{"
-                      << "    \"viewId\":" << view_id << ","
-                      << "    \"hitTestable\":true,"
-                      << "    \"focusable\":true"
-                      << "  }"
-                      << "}";
+  create_view_message << "{" << "  \"method\":\"View.create\","
```
Comment (loic-sharma) [inline shell/platform/fuchsia/flutter/tests/platform_view_unittest.cc:806]: Same comment for JSON string readability in this file

**Pull request merged**